debug/gdbserver.py

```diff
@@ -1649,14 +1649,16 @@ def setup(self):
         # pylint: disable=attribute-defined-outside-init
         self.gdb.load()
 
-        misa = self.hart.misa
+        # It may be affected by reset halt
+        misa = self.gdb.p(f"$misa=0x{self.hart.misa:x}")
```

Comment:
I don't quite get it. Which part is responsible for setting `misa.H`?

Reply:
When the power is turned on, the hart starts executing the brom code.

Reply:
When the power is turned on, the hart starts executing the brom code. Here does not affect this case, repeated testing will affect CheckMisa.